src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -383,11 +383,7 @@ void RegisterAppInterfaceRequest::Run() {
   // Version negotiation
   utils::SemanticVersion module_version(
       major_version, minor_version, patch_version);
-  if (mobile_version < utils::rpc_version_5) {
-    // Mobile versioning did not exist for
-    // versions before 5.0
-    application->set_msg_version(utils::base_rpc_version);
-  } else if (mobile_version < module_version) {
```

Comment:
This logic should not be removed, the reason for which is in the comment:
```
    // Mobile versioning did not exist for
    // versions before 5.0
```
The proxies did not send an appropriate message version in their RAI before mobile versioning was introduced (I believe that iOS sent 1.0), so this logic was introduced to account for that.

Reply:
@jacobkeeler fixed in 0edc0fac733780419b36f487213defd7166340f7